File: samples/hello-py/hello.py
import sys
import argparse

parser = argparse.ArgumentParser()
parser.add_argument("--name", required=True)
args = parser.parse_args()

print("╔════════════════════════════════════╗")
print(f"║  Hello {args.name: <26} ║")
print("║  from KODKAFA Python Plugin!       ║")
print("╚════════════════════════════════════╝")

File: samples/hello-r/hello.R
args <- commandArgs(trailingOnly = TRUE)
name_val <- "RUser"

if (length(args) > 1 && args[1] == "--name") {
  name_val <- args[2]
}

cat("╔════════════════════════════════════╗\n")
cat(sprintf("║  Name: %-27s ║\n", name_val))
cat("║  Hello from KODKAFA R Plugin!      ║\n")
cat("╚════════════════════════════════════╝\n")
